modules/lvms-scaling-storage-of-clusters-using-web-console.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-scaling-storage-of-clusters-using-web-console_{context}"]
+= Scaling up the storage of clusters by using the web console
```

Comment:
Same comment about dropping "by" if you want:
```suggestion
= Scaling up the storage of clusters using the web console
```

Reply:
I think it would be best to use "by" to keep it consistent with other titles in the doc.